modules/prometheus/group.go

```diff
@@ -0,0 +1,149 @@
+package prometheus
+
+import (
+	"strconv"
+	"strings"
+
+	"github.com/netdata/go.d.plugin/pkg/prometheus"
+	"github.com/prometheus/prometheus/pkg/labels"
+)
+
+type grouping interface {
+	chartID(pm prometheus.Metric) string
+	dimID(pm prometheus.Metric) string
+	dimName(pm prometheus.Metric) string
+}
```

Comment:
I don't really like this interface name or its descriptors. It feels very "un"-Go like 🤣

Reply:
i dont like it either 😆